### DISCUSSION TITLE
想问一下conntrack这个模块开启的作用，以及关闭能得到多少的性能提升

### DISCUSSION THREAD

**Discussion by zcl1115** [harmonycloud/kindling#605] [agent collector] [answered]
如题，想问一下爱conntrack这个模块开启的作用，以及关闭可以释放多少内存占用

**Comment by dxsup** [comment, marked as answer]
pod之间通过service访问时，通过conntrack找到对端pod信息。
通常这个模块不会有非常大的内存占用。

**Reply by zcl1115** [reply]
我们这边暂时还不会涉及到pod的对端访问，我这边关闭试一下

**Reply by dxsup** [reply]
只要通过service访问，我们都期望能帮用户看到这次调用访问到了哪个pod上，所以会使用conntrack。如果不需要看到调用了哪个pod，那是可以关闭的，关闭后只能看到pod->service的信息。
